ushahidi/platform-client-mzima
# Adding a language switch to mobile

This PR adds a translation-UI to the mobile-app, maps strings to translation-files and some translations to Spanish.

Note: The UI is a bit different than the designs in the ticket, this was done in conversation with Reuben.


To test:
1. Go to mobile app
2. Go to profile
3. Tap "Language"
4. Select Spanish
- [x] The radio-button is filled and the text "Current" moves to Spanish
Go out of Language and click on "Log in or Sign up"
- [x] Note that some of the UI is translated


Note 1: Not all strings are translated yet. 
Note 2: A lot of files are changed in this PR! I have noted in the code which ones to focus on, the rest is "just" the language-strings.
